packages/cookie_jar_conformance_tests/pubspec.yaml

```diff
@@ -0,0 +1,26 @@
+name: cookie_jar_conformance_tests
+description: >-
+  A library that tests whether implementations of package:cookie_jar's `CookieJar`
+  class behave as expected.
+repository: https://github.com/dart-lang/http/tree/master/pkgs/http_client_conformance_tests
+publish_to: none
+
+environment:
+  sdk: ^3.0.0
+
+dependencies:
+  cookie_store:
+    git:
+      url: https://github.com/nextcloud/neon
+      path: packages/cookie_store
+  meta: ^1.0.0
+  test: ^1.21.2
+  timezone: ^0.9.2
+  universal_io: ^2.0.0
+
+dev_dependencies:
+  lints: ^3.0.0
```

Comment:
```suggestion
```

Reply:
I'd love to also contribute this upstream so it was written in a neon independent way.

Reply:
I don't like the mix of things that is going on here. Separating the code into a package is good to ease upstreaming, but everything else should be according to our standards.